gazebo_ros_control:
  pid_gains:
    panda_joint1: {p: 100000, i: 0, d: 0}
    panda_joint2: {p: 20000, i: 0, d: 1000}
    panda_joint3: {p: 4000, i: 0, d: 0}
    panda_joint4: {p: 6000, i: 0, d: 120}
    panda_joint5: {p: 2000, i: 0, d: 80}
    panda_joint6: {p: 24000, i: 0, d: 40}
    panda_joint7: {p: 1000, i: 0.5, d: 0}
    panda_finger_joint1: {p: 100, i: 0, d: 50}
    panda_finger_joint2: {p: 100, i: 0, d: 50}